sections/event-types.txt

```diff
@@ -2236,6 +2237,19 @@ myDiv.addEventListener("auxclick", function(e) {
 			EVENT{wheel} event with multiple non-zero axes or as separate
 			EVENT{wheel} events for each non-zero axis.
 
+			A <a>user agent</a> SHOULD create a wheel event group when the first
+			wheel event is fired, so that all subsequent wheel events within a
+			implementation-specific amount of time can be targetted at the same
+			element.
+
+			<p class="example">
+			If a series of wheel events targetted in a scrollable element start
+			above a child element, later events for the same user gesture may
+			occur over the child element. In the case that {{Event/preventDefault()}}
+			is called in a wheel event handler for the child element, targetting
+			the child element could result in unexpected behavior for the user.
+			</p>
```

Comment:
Not sure if the example text is helpful as is, but I thought some sort of example might clarify things.

Reply:
I think having examples can be really useful.

I'm not sure what the takeaway is for this example, though. It ends with "could result in unexpected behavior", which makes it sound like a cautionary tale. Is this trying to explain why it is spec'ed this way, or is there something that the user needs to do to avoid the unexpected behavior?